sdk/scheduler/build.gradle

```diff
@@ -102,6 +104,13 @@ dependencies {
     compile "javax.el:javax.el-api:${elVer}"
     compile "org.glassfish.web:javax.el:${elVer}"
     compile 'com.mesosphere:mesos-http-adapter:0.4.1'
+    compile("com.auth0:java-jwt:${jwtVer}") {
+        // JWT package comes with jackson version 2.8.4 which doesn't work properly with
```

Comment:
I've actually tried to bump the library to the `2.8.4` version. The new version was a reason why loopback serialization wasn't working and hello-world placement integration tests were failing.. I've tried to debug it but didn't find a reason in a day so I've decided to keep existing version and possibly tackle the Jackson upgrade in a separate PR.